README.md

```diff
@@ -4,3 +4,38 @@
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+
+# This project was built with:
+
+
+- Html
+- Css
+- Javascript
+
+# Version Control System
+
+- GIT
+
+### To get a local copy up and running follow these simple steps.
+
+- You can clone this repo by typing `git clone https://github.com/prolajumokeoni/portfolio-mobile` on your terminal.
+
+- Type `cd portfolio-mobile-menu` to access the project on the terminal.
+
+## 👩🏿‍🏫Author
+### **Oni Olajumuke Priscilla**
+
+- Github [@githubhandle](https://github.com/prolajumokeoni)
+- Twitter [@twitterhandle](https://twitter.com/prolajumokeoni)
+- LinkedIn [@LinkedIn](https://www.linkedin.com/in/prolajumokeoni)
```

Comment:
### Optional

- [ ] Please kindly enter your account username at the different handle, We should have something like 
 Github [@prolajumokeoni](https://github.com/prolajumokeoni)